eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/db/sqlite/SqliteWalletDb.scala

```diff
@@ -197,13 +197,26 @@ object SqliteWalletDb {
 
   /**
     * change this value
-    * -if the new codec is incompatible with the old one
+    * - if the new codec is incompatible with the old one
     * - OR if you want to force a full sync from Electrum servers
     */
-  val version = 0x0000
+  val version = 0x0001
+
+  def walletTypeCodec(vers: Long): Codec[WalletType] = vers match {
```

Comment:
why not simply set version to `0x0001` and add new walletType field (a byte for example, 0 for P2SH_SEGWIT, 1 for BECH32, rest not defined yet) ?

Reply:
I thought a boolean was enough (after all is a binary choice here) but with the byte we are more _future-proof_, changes have been implemented in e5e6373